prov/gni/src/gnix_ep.c

```diff
@@ -360,15 +359,12 @@ static void __gnix_vc_destroy_ht_entry(void *val)
  * EP vc initialization helper
  ******************************************************************************/
 
-static inline int __gnix_ep_init_vc(struct gnix_fid_ep *ep_priv)
+int _gnix_ep_init_vc(struct gnix_fid_ep *ep_priv, enum fi_av_type type)
```

Comment:
This change doesn't make sense to me.  All the call sites just pass in `ep->av->type`.  Why is this desirable?